xml/systemd.xml

```diff
@@ -1485,51 +1483,78 @@ UMask=007</screen>
     <sect2 xml:id="sec-boot-systemd-custom-drop-in">
       <title>Creating drop-in files</title>
       <para>
-        For minor changes of a configuration file, use so-called drop-in files.
-        Drop-in files let you extend the configuration of unit files without
-        having to edit or override the unit files themselves.
+        For minor changes of a configuration file, use so-called drop-in files
+        or <emphasis>drop-ins</emphasis>. Drop-ins let you extend both unit and
+        daemon configuration files without having to edit or override the files
+        themselves. Drop-ins have higher precedence over main configuration
+        files. They are stored in the following directories:
       </para>
+      <variablelist>
+        <varlistentry>
+          <term><filename>/etc/systemd/*.conf.d/</filename></term>
+          <listitem>
+            <para>
+              Drop-ins added and customized by system administrators.
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><filename>/usr/lib/systemd/*.conf.d/</filename></term>
```

Comment:
Path for unit files:
/usr/lib/systemd/system/*.service.d/

See `systemd.unit(5)` for full _unit search path_.